packages/g/gpxsee/package.yml

```diff
@@ -10,12 +10,12 @@ summary    : a Qt-based GPS log file viewer and analyzer that supports all commo
 description: |
     a Qt-based GPS log file viewer and analyzer that supports all common GPS log file formats.
 builddeps  :
-    - pkgconfig(Qt5Positioning)
-    - pkgconfig(Qt5SerialPort)
-    - pkgconfig(Qt5Svg)
-    - pkgconfig(Qt5UiTools)
+    - pkgconfig(Qt6Positioning)
+    - pkgconfig(Qt6SerialPort)
+    - pkgconfig(Qt6Svg)
+    - pkgconfig(Qt6UiTools)
 setup      : |
-    %qmake CONFIG+=release gpxsee.pro PREFIX=%PREFIX%
+    qmake6 CONFIG+=release gpxsee.pro PREFIX=%PREFIX%
 build      : |
```

Comment:
Can you try build it with :
```
clang      : yes
optimize   :
    - speed
    - thin-lto
 ```
To make it in line with others qt6 packages

Reply:
Sounds reasonable. 

Want to do something useful too? Evaluate how current version of the package performs on bigger maps, like continent or country[1].  Later I'll update PR for you to evaluate if new upgrades and optimisations have desired effect. 

1. I don't have broadband for large downloads, but pre-rendered maps available [here](https://garmin.bbbike.org) 

Thanks